prefetch.bs

```diff
@@ -378,8 +378,23 @@ The user agent may [=prefetch record/cancel and discard=] records from the [=Doc
 
        <p class="note">An implementation might wish to do this if it believes that the prefetch will be consumed more than once. For example, if in the future the response is consumed by a prerender, that [=prerendering traversable=] might be [=destroy a top-level traversable|destroyed=] through various means. Normally that would mean the response is discarded, but if the implementation performs this step, then the prefetched response will still be available to serve a future navigation. [[PRERENDERING-REVAMPED]]
     1. If the user agent did not perform the previous optional step, then it must [=list/remove=] |record| from |navigable|'s [=navigable/active document=]'s [=Document/prefetch records=].
-
-
+    1. <span id="prefetch-activation-client-creation"></span>Let |reservedEnvironment| be a new [=environment=] with:
```

Comment:
How about using `create a reserved client` here (and perhaps merging the creation of `request` as well) to unify the code paths?
As far as I briefly checked, `request's reserved client` is unmodified since its creation here (but I might be missing something).

Reply:
Updated per offline discussion, PTAL!